internal/service/dns_config/model_config_acl.go

```diff
@@ -35,6 +36,8 @@ var ConfigACLAttrTypes = map[string]attr.Type{
 var ConfigACLResourceSchemaAttributes = map[string]schema.Attribute{
 	"comment": schema.StringAttribute{
 		Optional:            true,
+		Computed:            true,
```

Comment:
The field is computed because we set a default value?

Reply:
Yes.